### PR TITLE
Create experimental mt-pytorch code example

### PR DESCRIPTION
**Pull request by danieljanes**

### CODE REVIEW

File: examples/mt-pytorch/driver.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            req=get_results_req
+        )
+        # ------------------------------------------------------------------ Driver SDK
+
+        results: List[Result] = get_results_res.results
+        print(f"Got {len(get_results_res.results)} results")
+
+        all_results += results
+        if len(all_results) == len(task_ids):
+            break
```
Comment (tanertopal) [inline examples/mt-pytorch/driver.py:76]: @danieljanes We don't have an early stop here but as this is all experimental I am just noting my thoughts here :)

Comment (tanertopal) [reply examples/mt-pytorch/driver.py:76]: Oh, also would be cool to use `tqdm` here too.

**Review by tanertopal — APPROVED** [review]
As all of this is experimental I only left a note but apart from that lgtm.

File: examples/mt-pytorch/driver.py (hunk 1 above)
Comment (danieljanes) [reply examples/mt-pytorch/driver.py:76]: I agree that these things would be cool, but my goal was more to create a minimal example here, which is why I'd suggest to keep is focussed :) regarding the early stopping, my assumption was that the server would enforce a timeout on all tasks anyways, so the driver can just wait until it gets either a full result or something that tells it that a particular task did not finish in time.

**Review by tanertopal — APPROVED** [review]

**Pull request merged**